app/views/RegisterView/PasswordTips.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+
+import Tip from './Tip';
+import i18n from '../../i18n';
+import { useTheme } from '../../theme';
+import sharedStyles from '../Styles';
+
+const styles = StyleSheet.create({
+	PasswordTipsTitle: {
+		...sharedStyles.textMedium,
+		fontSize: 14,
+		lineHeight: 20
+	},
+	tips: {
+		gap: 8,
+		paddingTop: 8
+	}
+});
+
+interface IPasswordTips {
+	isDirty: boolean;
+	password: string;
+}
+
+const PasswordTips = ({ isDirty, password }: IPasswordTips) => {
+	const { colors } = useTheme();
+
+	const atLeastEightCharactersValidation = /^.{8,}$/;
```

Comment:
https://github.com/RocketChat/Rocket.Chat/blob/develop/packages/ui-contexts/src/hooks/useVerifyPassword.ts#L8

Reply:
got it, adjusted.